packages/oapp-alt-evm/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@layerzerolabs/oapp-alt-evm",
+  "version": "0.0.1",
+  "description": "LayerZero Labs reference EVM OmniChain Application (OAppAlt) for EndpointV2Alt implementation",
+  "keywords": [
+    "LayerZero",
+    "OmniChain",
+    "EndpointV2",
+    "EVM",
+    "OApp"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LayerZero-Labs/devtools.git",
+    "directory": "packages/oapp-alt-evm"
+  },
+  "license": "MIT",
+  "exports": {
+    "./package.json": "./package.json",
+    "./artifacts/*.json": {
+      "require": "./artifacts/*.json",
+      "imports": "./artifacts/*.json"
+    }
+  },
+  "files": [
+    "artifacts/**/*",
+    "contracts/**/*",
+    "test/**/*"
+  ],
+  "scripts": {
+    "build": "$npm_execpath compile",
```

Comment:
Please delete the `build` command: 

- `build` command builds JS artifacts and depends on `compile` (see `turbo.json`)
- `compile` command does any contract/program compilation